src/main/scala/devices/debug/Debug.scala

```diff
@@ -1159,3 +1418,89 @@ class DMIToTL(implicit p: Parameters) extends LazyModule {
 
   }
 }
+
+class SBToTL(implicit p: Parameters) extends LazyModule {
+
+  val node = TLClientNode(Seq(TLClientPortParameters(Seq(TLClientParameters("sba")))))
+
+  lazy val module = new LazyModuleImp(this) {
+    val io = IO(new Bundle {
+      val rdEn         = Bool(INPUT)
+      val wrEn         = Bool(INPUT)
+      val addrIn       = UInt(INPUT, 128) // TODO: Parameterize these widths
+      val sizeIn       = UInt(INPUT, 128)
+      val dataIn       = UInt(INPUT, 128)
+      val dmactive     = Bool(INPUT)
+      val dataOutValid = Bool(OUTPUT)
+      val dataOut      = UInt(OUTPUT, 128)
+      val sbStateOut   = UInt(OUTPUT, log2Ceil(SystemBusAccessState.maxId))
+    })
+
+    import uncore.devices._
+    import uncore.devices.DMI_RegAddrs._
```

Comment:
I'd consider adding `import SystemBusAccessState._` here to reduce clutter on the below lines.